src/rm_cms.c

```diff
@@ -54,72 +55,123 @@ static int CreateCMSKey(RedisModuleCtx *ctx, RedisModuleString *keyName,
 
 static int parseCreateArgs(RedisModuleCtx *ctx, RedisModuleString **argv, int argc,
                     long long *width, long long *depth) {
-    if(CompareCStringToRedisString("cms.initbydim", argv[0]) == 1) {
+    if(argc == 4) {
```

Comment:
You can use `args.{c,h}` from rmutil, might make it easier to parse

Reply:
used RMUtil_ArgIndex